flowauth/frontend/src/UserServer.jsx

```diff
@@ -39,10 +39,10 @@ class UserServer extends React.Component {
   getBody = () => {
     if (this.state.editing) {
       return (
-        <TokenDetails
+        <TokenBuilder
           classes={this.props.classes}
           onClick={this.toggleEdit}
-          serverID={this.props.serverID}
+          activeServer={this.props.serverID}
```

Comment:
Why rename only half of it?

Reply:
The serverID prop is used elsewhere, so I didn't want to touch it.